assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -296,7 +296,5 @@ export function getIconPath(icon: string, color: string = '') {
   return color ? `${ICONS_PATH}/${color}/${icon}` : `${ICONS_PATH}/${icon}`;
 }
 
-export const getRandomPictureUrl = () => {
-  const pictureId = Math.floor(Math.random() * PICTURES_LENGTH) + 1;
-  return `${PICTURE_BASE_URL}${pictureId}${PICTURE_EXTENSION}`;
-};
\ No newline at end of file
+// We `pictureId + 1` because there is no image in the S3 bucket with 0 as an id
```

Comment:
Then why does the id starts at 0?

Reply:
the picture id comes from the position of the fiction in an array: 
array[0] = first fiction => picture id should be 1 instead of 0 because no file on S3 with id 0
array[1] = second fiction and so on...